packages/cypress/.eslintrc.js

```diff
@@ -5,6 +5,7 @@ module.exports = {
       files: ['**/*.ts'],
       rules: {
         'import/no-extraneous-dependencies': ['error', { devDependencies: true }],
+        'import/no-unresolved': ['warn'],
```

Comment:
eslint was erroring on `import { archiveStorybook } from '@chromaui/shared-e2e/archive-storybook/scripts'`, although that seems to work fine with the type fixes from yesterday. Not sure how to fix this for real in eslint yet.